Frameworks/UIKit/UITableViewCell.mm

```diff
@@ -1482,6 +1482,12 @@ static void removeAllAnimationsFromLayers(CALayer* layer) {
     }
 }
 
+// determine if a UIView is internal subview used for building default chrome for UITableViewCell
+- (BOOL)_isInternalSubview:(UIView*)view {
+    return (view == _contentView || view == _curAccessoryView || view == _groupEdgeView || view == _removeButton ||
+            view == _editSelectedView || view == _backgroundView || view == _selectedbackgroundView || view == _borderView);
```

Comment:
nit: since half of these are removed/handled already in prepareForReuse, should we just do an explicit check for the remaining few in the for loop?